### ISSUE TITLE
20190412 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#49]
# 20190412 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [lib-pku/libpku](https://github.com/lib-pku/libpku) （北京大学课程资料整理）
* [stackshareio/awesome-stacks](https://github.com/stackshareio/awesome-stacks) （A curated list of tech stacks for building different applications &amp; features）
* [ctripcorp/CRN](https://github.com/ctripcorp/CRN) （CRN是Ctrip React Native简称，由携程无线平台研发团队基于React Native框架优化，定制成稳定性和性能更佳、也更适合业务场景的跨平台开发框架。）
* [LisaDziuba/Awesome-Design-Tools](https://github.com/LisaDziuba/Awesome-Design-Tools) （The best design tools for everything.）
* [521xueweihan/git-tips](https://github.com/521xueweihan/git-tips) （Git的奇技淫巧）
* [datawhalechina/pumpkin-book](https://github.com/datawhalechina/pumpkin-book) （《机器学习》（西瓜书）公式推导解析，在线阅读地址：https://datawhalechina.github.io/pumpkin-book）
* [vuejs/vue](https://github.com/vuejs/vue) （        A progressive, incrementally-adoptable JavaScript framework for building UI on the web.      ）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [lukeed/dimport](https://github.com/lukeed/dimport) （Run ES Module syntax (`import`, `import()`, and `export`) in any browser – even IE!）
* [mathdroid/crop-github-images-cli](https://github.com/mathdroid/crop-github-images-cli) （Crop image/gif into swag for your Github profile）
* [agalwood/Motrix](https://github.com/agalwood/Motrix) （A full-featured download manger.）
* [YMFE/yapi](https://github.com/YMFE/yapi) （YApi 是一个可本地部署的、打通前后端及QA的、可视化的接口管理平台）
* [trekhleb/javascript-algorithms](https://github.com/trekhleb/javascript-algorithms) （Algorithms and data structures implemented in JavaScript with explanations and links to further readings）
* [GoogleChrome/puppeteer](https://github.com/GoogleChrome/puppeteer) （Headless Chrome Node API）
* [30-seconds/30-seconds-of-code](https://github.com/30-seconds/30-seconds-of-code) （Curated collection of useful Javascript snippets that you can understand in 30 seconds or less.）
* [nefe/You-Dont-Need-jQuery](https://github.com/nefe/You-Dont-Need-jQuery) （Examples of how to do query, style, dom, ajax, event etc like jQuery with plain javascript.）
* [airbnb/javascript](https://github.com/airbnb/javascript) （        JavaScript Style Guide      ）
* [ljianshu/Blog](https://github.com/ljianshu/Blog) （一起学前端，个人博客合集，喜欢的请点 star，想订阅请点 watch）
* [statsbotco/cube.js](https://github.com/statsbotco/cube.js) （&#x1f4ca; Serverless Analytics Framework）
* [facebook/create-react-app](https://github.com/facebook/create-react-app) （Create React apps with no build configuration.）
* [axios/axios](https://github.com/axios/axios) （Promise based HTTP client for the browser and node.js）
* [storybooks/storybook](https://github.com/storybooks/storybook) （        UI Component Dev Environment for React      ）
* [mrdoob/three.js](https://github.com/mrdoob/three.js) （        JavaScript 3D library.      ）
* [nodejs/node](https://github.com/nodejs/node) （        Node.js JavaScript runtime ✨??✨      ）
* [gatsbyjs/gatsby](https://github.com/gatsbyjs/gatsby) （⚛️?? Blazing fast static site generator for React）
* [weilanwl/ColorUI](https://github.com/weilanwl/ColorUI) （鲜亮的高饱和色彩，专注视觉的小程序组件库）
* [leeoniya/dropcss](https://github.com/leeoniya/dropcss) （A simple, thorough and fast unused-CSS cleaner）
* [Molunerfinn/PicGo](https://github.com/Molunerfinn/PicGo) （A simple &amp; beautiful tool for pictures uploading built by electron-vue）

## Vue
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [dcloudio/uni-app](https://github.com/dcloudio/uni-app) （使用 Vue.js 开发跨平台应用的前端框架）
* [ClintOxx/vuetify-admin-dashboard](https://github.com/ClintOxx/vuetify-admin-dashboard) （A Crud Admin panel made from Vue js and Vuetify）
* [vueComponent/ant-design-vue](https://github.com/vueComponent/ant-design-vue) （An enterprise-class UI components based on Ant Design and Vue. ?）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [iview/iview](https://github.com/iview/iview) （        A high quality UI Toolkit with both Vue.js 2.0 and 1.0      ）
* [fengT-T/996_list](https://github.com/fengT-T/996_list) （996.ICU 和 996.ICU 的匿名投票列表）
* [sendya/ant-design-pro-vue](https://github.com/sendya/ant-design-pro-vue) （??‍???‍? Use Ant Design Vue like a Pro! Demo: https://pro.loacg.com | React: http://pro.ant.design/）
* [bestony/logoly](https://github.com/bestony/logoly) （A Pornhub Pattern Logo Generator）
* [airyland/vux](https://github.com/airyland/vux) （        Mobile web UI Components based on Vue and WeUI (Vue@2.x)      ）
* [elunez/eladmin-qd](https://github.com/elunez/eladmin-qd) （eladmin前端源码，项目基于 Spring Boot 2.1.0 、 Spring Boot Jpa、 Spring Security、Redis、Vue的前后端分离后台管理系统， 权限控制采用 RBAC，菜单动态路由）
* [heyui/heyui](https://github.com/heyui/heyui) （UI Toolkit for Web, Vue2.0）
* [LinusBorg/portal-vue](https://github.com/LinusBorg/portal-vue) （        A Portal Component for Vuejs, for rendering DOM outside of a component, anywhere in the document.      ）
* [Akryum/vue-virtual-scroller](https://github.com/Akryum/vue-virtual-scroller) （Smooth scroll with any amount of data）
* [ElemeFE/mint-ui](https://github.com/ElemeFE/mint-ui) （        Mobile UI elements for Vue.js      ）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [what-crud/vue-crud](https://github.com/what-crud/vue-crud) （Vue.js based REST-ful CRUD system）
* [DivineITLimited/hyper-editor](https://github.com/DivineITLimited/hyper-editor) （A backend independent visual composer for web.）

## HTML
* [QSCTech/zju-icicles](https://github.com/QSCTech/zju-icicles) （浙江大学课程攻略共享计划）
* [nndl/nndl.github.io](https://github.com/nndl/nndl.github.io) （《神经网络与深度学习》 Neural Network and Deep Learning）
* [jaywcjlove/linux-command](https://github.com/jaywcjlove/linux-command) （        Linux命令大全搜索工具，内容包含Linux命令手册、详解、学习、搜集。      ）
* [uswds/public-sans](https://github.com/uswds/public-sans) （A strong, neutral, principles-driven, open-source typeface for text or display）
* [MLH/mlh-hackathon-flask-starter](https://github.com/MLH/mlh-hackathon-flask-starter) （Hackathon starter project for Flask applications）
* [kognise/water.css](https://github.com/kognise/water.css) （A just-add-css collection of styles to make simple websites just a little nicer）
* [ColorlibHQ/AdminLTE](https://github.com/ColorlibHQ/AdminLTE) （AdminLTE - Free Premium Admin control Panel Theme Based On Bootstrap 3.x）
* [google/styleguide](https://github.com/google/styleguide) （        Style guides for Google-originated open-source projects      ）
* [JokerJohn/BilibliSource](https://github.com/JokerJohn/BilibliSource) （B站视频部分课件）
* [lib-pku/lib-pku.github.io](https://github.com/lib-pku/lib-pku.github.io) （北京大学课程资料整理）
* [iliakan/javascript-tutorial-en](https://github.com/iliakan/javascript-tutorial-en) （        Modern JavaScript Tutorial       ）
* [fengdu78/Coursera-ML-AndrewNg-Notes](https://github.com/fengdu78/Coursera-ML-AndrewNg-Notes) （吴恩达老师的机器学习课程个人笔记）
* [jimmywarting/StreamSaver.js](https://github.com/jimmywarting/StreamSaver.js) （StreamSaver writes stream to the filesystem directly asynchronous）
* [ColorlibHQ/gentelella](https://github.com/ColorlibHQ/gentelella) （Free Bootstrap 3 Admin Template）
* [cs231n/cs231n.github.io](https://github.com/cs231n/cs231n.github.io) （        Public facing notes page      ）
* [wesbos/JavaScript30](https://github.com/wesbos/JavaScript30) （        30 Day Vanilla JS Challenge      ）
* [learn-awesome/learn-awesome](https://github.com/learn-awesome/learn-awesome) （Curated lists of awesome learning resources on various topics across formats.）

## CSS
* [twbs/bootstrap](https://github.com/twbs/bootstrap) （The most popular HTML, CSS, and JavaScript framework for developing responsive, mobile first projects on the web.      ）
* [tailwindcss/tailwindcss](https://github.com/tailwindcss/tailwindcss) （A utility-first CSS framework for rapid UI development.）
* [uswds/uswds](https://github.com/uswds/uswds) （The U.S. Web Design System helps the federal government build fast, accessible, mobile-friendly websites.）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [ryanoasis/nerd-fonts](https://github.com/ryanoasis/nerd-fonts) （? Iconic font aggregator, collection, and patcher: 39 patched fonts (87k+ variations). Includes popular glyph collections such as Font Awesome &amp; fonts such as Hack）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [zhaoolee/StarsAndClown](https://github.com/zhaoolee/StarsAndClown) （☀️Github星聚弃疗榜,为Github优秀项目写一封推荐信，让Github优秀项目造福人类~ Github apocalypse list, write a letter of recommendation for Github's outstanding projects, and let Github excellent projects benefit humanity~）
* [TalkingData/iview-weapp](https://github.com/TalkingData/iview-weapp) （一套高质量的微信小程序 UI 组件库）
* [Microsoft/TypeScript-Node-Starter](https://github.com/Microsoft/TypeScript-Node-Starter) （        A starter template for TypeScript and Node with a detailed README describing how to use the two together.      ）
* [luangjokaj/wordpressify](https://github.com/luangjokaj/wordpressify) （? A build system designed to automate your WordPress development workflow.）
* [troxler/awesome-css-frameworks](https://github.com/troxler/awesome-css-frameworks) （List of awesome CSS frameworks）
* [StylishThemes/GitHub-Dark](https://github.com/StylishThemes/GitHub-Dark) （         Dark GitHub style      ）
* [github/personal-website](https://github.com/github/personal-website) （Code that'll help you kickstart a personal website that showcases your work as a software developer.）
* [rstacruz/cheatsheets](https://github.com/rstacruz/cheatsheets) （My cheatsheets）
* [twbs/rfs](https://github.com/twbs/rfs) （Automated responsive font sizes）
* [Requarks/wiki](https://github.com/Requarks/wiki) （Wiki.js | A modern, lightweight and powerful wiki app built on NodeJS, Git and Markdown）
* [apachecn/hands-on-ml-zh](https://github.com/apachecn/hands-on-ml-zh) （&#x1f4d6; [译] Sklearn 与 TensorFlow 机器学习实用指南）
* [Remix-Design/remixicon](https://github.com/Remix-Design/remixicon) （Open source neutral style icon system）
* [getgridea/gridea](https://github.com/getgridea/gridea) （✍️一个静态博客写作客户端 (A static blog writing client)）
* [d2l-ai/d2l-ko](https://github.com/d2l-ai/d2l-ko) （한글 번역이 진행 중 입니다 | Dive into Deep Learning. With code, math, and discussions.）

## TypeScript
* [codex-team/editor.js](https://github.com/codex-team/editor.js) （A block-styled editor with clean JSON output）

## Unknow
* [gothinkster/realworld](https://github.com/gothinkster/realworld) （        A Collection &amp; Specification for Exemplary Frontend and Backend Codebases ?      ）
* [h5bp/Front-end-Developer-Interview-Questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions) （        A list of helpful front-end related questions you can use to interview potential candidates, test yourself or completely ignore.      ）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [加我微信，进500人微信群，领100本前端电子书](http://t.cn/EV7pcM1)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
